java/ql/src/change-notes/2022-08-18-android-allowbackup-query.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* Added a new query, `java/android/allowBackup-attribute-enabled`, to detect if the `android:allowBackup` attribute is enabled in the Android manifest.
```

Comment:
I think after the recent changes this needs rewording:

```suggestion
* Added a new query, `java/android/backup-enabled`, to detect if Android applications allow backups.
```

Note that the ID also changed (see my other comment).